assets/scss/bootstrap-migration/custom/_buttons.scss

```diff
@@ -0,0 +1,3 @@
+.btn {
+    @include button-variant-disabled();
```

Comment:
Mixing is needed only if it will reused. You may just place this code here

Reply:
done